### PR TITLE
feature: ✨ access wallet button merge

### PR DESCRIPTION
**Pull request by NewCodeOnDaBlock**
### Feature

* \[x] Added entry to ./changelog
* \[x] Add test cases or procedure
* \[x] Add PR label

### CODE REVIEW

**Review by Abyscuit — APPROVED** [review]

**Review by andrewdahn — APPROVED** [review]

File: src/modules/access-wallet/ModuleAccessWalletHardware.vue [hunk 1]
```diff
@@ -198,7 +198,7 @@
           LedgerX
         =====================================================================================
         -->
-      <access-wallet-ledger-x
+      <!-- <access-wallet-ledger-x
```
Comment (gamalielhere) [inline src/modules/access-wallet/ModuleAccessWalletHardware.vue:201]: remove unused code.

File: src/modules/access-wallet/ModuleAccessWalletHardware.vue [hunk 2]
```diff
@@ -301,17 +301,16 @@ export default {
...
-          type: WALLET_TYPES.LEDGER
-        },
-        {
-          label: 'Ledger Bluetooth',
+          label: 'Ledger',
           icon: require('@/assets/images/icons/hardware-wallets/Ledger-Nano-X-Label-Icon.svg'),
-          ble: true,
           type: WALLET_TYPES.LEDGER
         },
+        // {
```
Comment (gamalielhere) [inline src/modules/access-wallet/ModuleAccessWalletHardware.vue:308]: remove unused code.

File: src/modules/access-wallet/ModuleAccessWalletHardware.vue [hunk 3]
```diff
@@ -769,8 +773,8 @@ export default {
         .then(_hwWallet => {
           try {
             this.loaded = true;
-            if (this.onLedger) this.ledgerConnected = true;
-            if (this.onLedgerX) this.nextStep();
+            // if (this.onLedger) this.ledgerConnected = true;
```
Comment (gamalielhere) [inline src/modules/access-wallet/ModuleAccessWalletHardware.vue:776]: remove unused code.

File: src/modules/access-wallet/hardware/components/AccessWalletLedger.vue [hunk 4]
```diff
@@ -95,9 +90,9 @@ export default {
     };
   },
   computed: {
-    btnTitle() {
-      return this.ledgerConnected ? 'Unlock Ledger' : 'Connect Ledger';
-    }
+    // btnTitle() {
```
Comment (gamalielhere) [inline src/modules/access-wallet/hardware/components/AccessWalletLedger.vue:93]: remove unused code.

File: src/modules/access-wallet/hardware/components/AccessWalletLedgerX.vue [hunk 5]
```diff
@@ -95,11 +103,11 @@ export default {
     };
   },
   computed: {
-    btnTitle() {
-      return this.ledgerConnected
-        ? 'Unlock Ledger Nano X'
-        : 'Connect Ledger Nano X';
-    }
+    // btnTitle() {
```
Comment (gamalielhere) [inline src/modules/access-wallet/hardware/components/AccessWalletLedgerX.vue:106]: remove unused code.

**Review by gamalielhere — CHANGES_REQUESTED** [review]

**Pull request merged**